vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -51,7 +51,16 @@ def save_container_groups_inventory(ems, hashes, target = nil)
                 []
               end
 
-    save_inventory_multi(:container_groups, ems, hashes, deletes, [:ems_ref], [:container_definitions, :containers, :labels])
+    hashes.each do |h|
+      next if h[:container_node_name].nil?
+      node = ContainerNode.first(
+        :conditions => {:ems_id => ems.id, :name => h[:container_node_name]})
+      h[:container_node_id] = node.nil? ? nil : node.id
```

Comment:
@simon3z yeah, do that!

Assign the `container_node_id` key, and be sure to include the `:container_node` key in the last array in the method call (like you're doing already).